code/modules/virus2/items_devices.dm

```diff
@@ -26,7 +26,7 @@
 /obj/item/weapon/virusdish
 	name = "virus containment/growth dish"
 	icon = 'icons/obj/items.dmi'
-	icon_state = "implantcase-b"
+	icon_state = "implantcase-blue"
```

Comment:
people probably are not expecting their virus dishes to change color

Reply:
those are the same, I renamed it for clarity